.github/workflows/linux-eic-shell.yml

```diff
@@ -8,18 +8,6 @@ on:
       - '*'
   pull_request:
   merge_group:
-  workflow_dispatch:
```

Comment:
I'd keep this one line, and remove the dependency repository info. Just having a workflow dispatch trigger gives us a run button at https://github.com/eic/epic/actions/workflows/linux-eic-shell.yml which has been useful at times.

Reply:
Sure, added with an inline comment.